Documentation/ApiOverview/GlobalValues/Constants/Index.rst

```diff
@@ -88,12 +88,8 @@ File types
 ..  versionchanged:: 13.0
```

Comment:
```suggestion
..  versionchanged:: 14.0
```

Reply:
You could write ...has been introduced with TYPO3 v13.0. 

It is a versionchanged directive not a versionadded directive. Also version hints for 14.0 will be kept longer then those for 13.0

Reply:
Should be a versionadded in the first place IMHO.

Reply:
In my eyes it should have been a deprecation. But it doesnt matter this much, lets move forward